pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -76,6 +76,12 @@ func (ctrl *Controller) syncNodeConfigHandler(key string) error {
 	if err := ctrl.cleanUpDuplicatedMC(managedNodeConfigKeyPrefix); err != nil {
 		return err
 	}
+	// explicitly setting the cgroupMode to "v2" and also updating the config node's spec if found empty
+	// This helps in updating the cgroupMode on all the worker nodes if they still have cgroupsv1 (Ex: RHEL8 workers)
+	if nodeConfig.Spec.CgroupMode == emptyInput {
```

Comment:
nit: there's a `CgroupModeEmpty` we could use here instead

Reply:
Addressed